components/supervisor-api/status.proto

```diff
@@ -122,8 +123,8 @@ message PortsStatusResponse {
 }
 
 enum PortVisibility {
-    private = 0;
-    public = 1;
+    privateVisibility = 0;
+    publicVisibility = 1;
```

Comment:
```suggestion
    private_visibility = 0;
    public_visibility = 1;
```

The camelCase doesn't transport on the Go side.
That said, what motivates that change?

Reply:
Motivation:

> PortVisibility is status.proto uses reserved keywords (public and private) as enum values that breaks the Java classes.

(see ToDo above)

For now, it's a temporary fix. Still need a good way to handle this problem. Suggestions welcome. Thought about just patching the generated Java files after generating.

Reply:
There probably some proto annotations which can drive java generation?